plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/ast/Node.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.viper.ast
+
+/**
+ * Class wrapper for Viper's AST nodes.
+ */
+data class Node(val node: viper.silver.ast.Node) : IntoSilverCallable by delegateToSilverCallable(node)
```

Comment:
What exactly is a Node?  Is it different to an Exp?  More general?

Reply:
See what's best more broadly; do other errors sometimes refer to statements?

Reply:
Oh yeah, we have one case where we obtain statements! When we are dealing with `MethodCall`s. So, I prefer sticking with the generic `Node` class. Is that ok?